StarkInfra/StarkInfra/DynamicBrcode/Discounts.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using StarkInfra.Utils;
+
+
+namespace StarkInfra
+{
+    /// <summary>
+    /// Discounts object
+    /// <br/>
+    /// Invoice discount information.
```

Comment:
Esse objeto Discount não é para Invoice. A descrição seria `Used to define a Discount in the BR Code`